### PR TITLE
Use Qt6 for Fedora 39 or newer

### PR DESCRIPTION
**Pull request by runlevel5**
Majority of Fedora 39 are now built with Qt6.

### CODE REVIEW

File: vcmi.spec [hunk 1]
```diff
@@ -43,8 +43,14 @@ BuildRequires:  minizip-ng-devel
 BuildRequires:  tbb-devel
 BuildRequires:  zlib-devel
 BuildRequires:  ffmpeg-devel
+%if 0%{?fedora} == 39
```
Comment (kwizart) [inline vcmi.spec]: It should be >= so it can handle newer fedora releases (rawhide is currently at 40).

Once fixed (and tested) feel free to apply.

Comment (runlevel5) [reply vcmi.spec]: I do not know if rawhide and 40 would be having the same package name hence I deliberately target only 39 right now

Comment (kwizart) [reply vcmi.spec]: Then you are wrong twice ;)
You are expected to land newer features in the most recent branch. So please verify rawhide, then f39

**Comment by runlevel5** [comment]
@leigh123linux @kwizart please help me review this PR. Many thanks

**Comment by kwizart** [comment]
Fine with me.
Please merge in our upstream remote and build for f39 and f40. If anything please ask.
Thanks

**Comment by runlevel5** [comment]
@kwizart I don't have permission to perform the merge.

Waiting for people who do to help me with that task. Many thanks

**Comment by leigh123linux** [comment]
> @kwizart I don't have permission to perform the merge.
> 
> Waiting for people who do to help me with that task. Many thanks

You/We don't merge here.
github is just a mirror of our git instance,

https://pkgs.rpmfusion.org/cgit/free/vcmi.git/

See this for more info.  https://rpmfusion.org/Contributors#Install_RPMFusion_packager_tools_and_set_your_environment

```
rfpkg co free/vcmi
```

**Pull request closed without merging**